### ISSUE TITLE
I can't define 'searchFilter' using the File provider

### ISSUE THREAD

**Issue by c0rzair1** [wiltonsr/ldapAuth#9]
I can't define **searchFilter** in _dynamic configuration_ when using the _File provider_.
When I set the config like:

```
http:
  routers:   ...
  services:  ...

  middlewares:
    my-ldapAuth:
      plugin:
        ldapAuthPlugin:
          Enabled: true
          LogLevel: DEBUG
          Url: ldap://someserver
          Port: 389
          BaseDN: DC=com
          BindDN: CN=ldapuser,OU=HUB,DC=com
          BindPassword: Pa$$w0rd
          Attribute: sAMAccountName
          SearchFilter: (&(objectClass=user)(!(objectClass=computer))({{.Attribute}}={{.Username}}))         
```

Then when I run T**raefik** I get the error:
```
traefik    | time="2022-01-20T10:04:58Z" level=info msg="Starting provider *file.Provider {\"watch\":true,\"filename\":\"/configurations/dynamic.yml\"}"
traefik    | time="2022-01-20T10:04:58Z" level=error msg="Cannot start the provider *file.Provider: template: /configurations/dynamic.yml:45:72: executing \"/configurations/dynamic.yml\" at <.Attribute>: can't evaluate field Attribute in type bool"
```

As a workaround, I specified my filter as the default value of an option **SearchFilter** in the file **`ldapauth.go`**:
```
  ...
// CreateConfig creates the default plugin configuration.
func CreateConfig() *Config {
        return &Config{
                Enabled:        true,
                       ...
                SearchFilter:   "(&(objectClass=user)(!(objectClass=computer))({{.Attribute}}={{.Username}}))",
                       ...
                Username:     "",
        }
}
```

**Comment by wiltonsr** [comment]
Hi @c0rzair1 

I fixed this issue and release `v0.0.13`. 

If you are using `Pilot`, please wait a while to reflect this in [Plugin page](https://pilot.traefik.io/plugins/61560239fbe8e5d6fa495126/ldap-auth) and adjust the version in your conf.

If still there is any problem, please let me know.

**Comment by dorianim** [comment]
Hi @wiltonsr 

I just wanted to mention, that this also works:
```yml
 searchFilter: ({{"{{.Attribute}}={{.Username}}"}})
```
Looks a bit cleaner to me :)

Regards,
Dorian
